shuup/simple_cms/admin_module/views.py

```diff
@@ -159,6 +181,15 @@ def save_form(self, form):
     def get_queryset(self):
         return super(PageEditView, self).get_queryset().for_shop(get_shop(self.request)).filter(deleted=False)
 
+    def get_toolbar(self):
```

Comment:
This method has the same implementation of its super, hence, it's not needed here. If you want to add provides entry point here this could makes sense, but the logic for that is missing.